docs/changelog.rst

```diff
@@ -4,6 +4,11 @@ Changelog
 0.29.0 (January 2024)
```

Comment:
```suggestion
0.30.0 (January 2024)

Reply:
Not sure if this is required, we will clean this up before doing a release anyway. Feel free to merge without this.